src/hera.cpp

```diff
@@ -50,6 +52,8 @@ using namespace HeraVM;
 enum hera_evm_mode {
   EVM_FALLBACK,
   EVM2WASM,
```

Comment:
Maybe rename this to EVM2WASM_CONTRACT for clarity

Reply:
Do you want to create a PR for that?

Reply:
@axic #235